File: hellomac/CounterApp/CounterApp/ViewController.m
//
//  ViewController.m
//  CounterApp
//
//  Created by Takayasu Sakai on 2015/11/26.
//  Copyright © 2015年 Takayasu Sakai. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController {
    int kazu;
    NSString *kazutext;
    IBOutlet UILabel *kazulabel;
}

- (void)viewDidLoad {
    [super viewDidLoad];
    kazu = 0;
}

- (IBAction)plusup:(id)sender {
    kazu++;
    kazutext = [NSString stringWithFormat:@"%d", kazu];
    [kazulabel setText:kazutext];
}

- (IBAction)mainusdown :(id)sender {
    if( kazu > 0){
        kazu--;
    }
    kazutext = [NSString stringWithFormat:@"%d", kazu];
    [kazulabel setText:kazutext];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: hellomac/Slidebar/Slidebar/ViewController.swift
//
//  ViewController.swift
//  Slidebar
//
//  Created by Takayasu Sakai on 2015/12/13.
//  Copyright © 2015年 Takayasu Sakai. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var label: UILabel!
    @IBAction func showValue(sender: UISlider) {
        label.text = "\(sender.value)"
    }
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

